WISE-Community/WISE
# Properly size images on site

Optimized images on the site by compressing and resizing based on actual screen display pixels. Now offering 1x and 2x sizes for most images so that low and high res screens download separate files. Also serving `.webp` files for browsers that support the format.

To test:
- Load site and visit homepage, `/about`, `/features`, `/login`, `/register`.
- Make sure images all load properly on each route.
- Check to see that `.webp` files are being loaded (if browser supports) and that 2x size images are being loaded for high res screens.

Closes #2338.
